Setup the My Portfolio project and coding the main mobile page.

paulsaenzsucre opened the pull request:
In this pull request a made these changes:

- Write an index.html file with a toolbar(header) and a headline (main) using HTML5.
- Write a styles.css file with several styles for the toolbar and headline. Use Flexbox to lay out the elements and pseudo-classes to implement some basic interactions.
- Write a descriptive README file.
- Include a MIT license.

Also, did the following activities:

- Setup the linters for my project.
- Look for and fix linter errors.
- Create a new branch for coding new features.
- Do frequents commits with incremental changes.

paulsaenzsucre committed:
Add linter.yml in setup-and-mobile-first branch

-Add linters in setup-and-mobile-first new branch.

paulsaenzsucre committed:
Setup linters in my local enviroment.

- Add node_modules\ to .gitignore.

- Add .hintrc to root directory.

Add .stylelintrc.json to root directory.

Add package-lock.json to root directory.

Add package.json to root directory.

paulsaenzsucre committed:
Modifying README template for the project.

paulsaenzsucre committed:
Setup a basic HTML|CSS project.

- Add a index file using a HTML5 template.

- Add a style.css file, with a basic style for body tag, in a css folder.

- Link style.css to index.html file.

- Add .vscode to .gitignore file.

paulsaenzsucre committed:
Add images files to image folder.

- Add social media icons files: LinkedLinkedin.svg, angellist.svg, github.svg, medium.svg, twitter.svg.

Add hamburger menu icon file: hamburger.svg.

Add background main image: header-illsutration-mobile.svg.

paulsaenzsucre committed:
Add toolbar HTML and CSS.

- Add html for toolbar: header tag with h2 and img tags.

Add styles for toolbar: use Flexbox to lay out the elements, use pseudo-classes to implement interactions (hover).

paulsaenzsucre committed:
Add headline HTML and CSS.

- Add html for headline:  main tag with h1, p and div tags; also five a tags containing five social media icons in img tags.

- Add styles for headline: use Flexbox to lay out the elements, use pseudo-classes to implement interactions (hover).

paulsaenzsucre committed:
Fix linter errors in styles.css

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,33 @@
+</head>
+<body>
+  <header id="toolbar">
+    <h2 id="logoPlaceHolder">Welcome</h2>
```
Kindly make this logo a link by wrapping it around an a tag. This is the best convention and also good for user experience.

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,33 @@
+<body>
+  <header id="toolbar">
+    <h2 id="logoPlaceHolder">Welcome</h2>
+    <img id="hamburgerButton" src="images/hamburger.svg" alt="Hamburger button">
```
For semantic reasons, kindly put the hamburger menu in a nav tag.

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,33 @@
+      <a href="https://medium.com/"><img src="images/medium.svg" alt="Medium logo" class="social-media-icons"></a>
+      <a href="https://github.com/elafildecolor"><img src="images/github.svg" alt="Github logo" class="social-media-icons"></a> 
+      <a href="https://www.angellist.com/"><img src="images/angellist.svg" alt="Angellist logo" class="social-media-icons"></a> 
+    </div>
```
Kindly wrap this in a section tag since it is a standalone section of the document. :+1:

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,33 @@
+      <a href="https://www.linkedin.com/feed/"><img src="images/Linkedin.svg" alt="Linkedin logo" class="social-media-icons"></a>
+      <a href="https://medium.com/"><img src="images/medium.svg" alt="Medium logo" class="social-media-icons"></a>
+      <a href="https://github.com/elafildecolor"><img src="images/github.svg" alt="Github logo" class="social-media-icons"></a> 
+      <a href="https://www.angellist.com/"><img src="images/angellist.svg" alt="Angellist logo" class="social-media-icons"></a> 
```
The list of icons should use a ul tag to group them instead of the div tag used here, and li per icon; each icon has a link (a tag). Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href.

jerryowusu commented on css/styles.css:
```diff
@@ -0,0 +1,90 @@
+
+#hamburgerButton:hover {
+  transform: scale(1.2);
+  transition: transform 0.2s;
```
Kindly ensure that your design matches that of the figma in all screen sizes of the mobile version ie. with below 765px. There are some disparities between your design and the figma design as shown below. Kindly check the width, spacing, height, etc and fix accordingly.

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-28 06-50-28](https://user-images.githubusercontent.com/65233860/198522719-733df70f-796e-449f-8e38-750258e93d30.png)  |  ![Screenshot from 2022-10-28 06-55-34](https://user-images.githubusercontent.com/65233860/198522940-25de2d3f-7eba-4363-8f5b-5360b05e7f12.png)

jerryowusu reviewed (changes requested):
Hi @elafildecolor 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

![so-close](https://user-images.githubusercontent.com/65233860/198523577-6d5bb4e2-d7cb-4da2-bb6a-e1b49ff4325f.gif)


## Highlights
All linters passing :heavy_check_mark: 
Good commit messages :heavy_check_mark: 
Followed GitHub flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

paulsaenzsucre committed:
Fix issues in the code review.

- Do the logo placeholder a link wrapping around an <a> tag.

- Put the hamburger menu in a <nav> tag.

- Wrap the headline in a <section> tag.

- Put the social media icon inside <li> tags inside <ul> tag.

Making design match figma in all screen sizes below 765px.

Fixing some typos in the code.

zieeco reviewed (approved):
# status: APPROVED :green_circle: :100: 

Hi, @elafildecolor , 


![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966](https://user-images.githubusercontent.com/85113567/164945882-0a9554ea-956c-4fa0-a95c-379285283c1d.gif)


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
